SDKs/PowerShell/JumpCloud.SDK.V2/custom/generated/Set-JcSdkSystemAssociation.ps1

```diff
@@ -48,13 +48,22 @@ curl -X POST https://console.jumpcloud.com/api/v2/systems/{System_ID}/associatio
   }'
 ```
 .Example
-PS C:\> {{ Add code here }}
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'add' -type 'user' -Attributes @{ 'sudoEnabled' = $true; 'SudoWithoutPassword' = $true }
 
-{{ Add output here }}
 .Example
-PS C:\> {{ Add code here }}
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'update' -type 'user' -Attributes @{ 'sudoEnabled' = $true; 'SudoWithoutPassword' = $true }
 
-{{ Add output here }}
+.Example
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserID> -op 'remove' -type 'user' -Attributes @{ 'sudoEnabled' = $false; 'SudoWithoutPassword' = $false }
+
+.Example
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserGroupID> -op 'add' -type 'user_group' -Attributes @{ 'sudoEnabled' = $true; 'SudoWithoutPassword' = $false }
+
+.Example
+PS C:\> Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserGroupID> -op 'update' -type 'user_group' -Attributes @{ 'sudoEnabled' = $true; 'SudoWithoutPassword' = $false }
+
+.Example
```

Comment:
same thing here, there's no need to pass in attributes when you remove a system association we can just do:

`Set-JcSdkSystemAssociation -SystemId <SystemID> -id <UserGroupID> -op 'remove' -type 'user_group'`